recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 103b4e345d8cddbb5bb90940216d983a086a668546634d06658addcfdbd9a192
 
 build:
-  number: 3
+  number: 5
```

Comment:
#21 updates this to `4`, and we'll wait to merge this PR until #21 has been merged.